ui/component/or-mwc-components/src/or-mwc-input.ts

```diff
@@ -125,7 +125,8 @@ export enum InputType {
     CRON = "cron",
     DURATION = "duration",
     DURATION_TIME = "duration-time",
-    DURATION_PERIOD = "duration-period"
+    DURATION_PERIOD = "duration-period",
+    FILTER = "filter"
```

Comment:
Don't think we want `filter` as an input type; for background, `or-mwc-input` is intended to be a simple component that just applies material design styles to basic input fields.

Filter here is being used to apply some styling which should be achievable by just using `InputType#TEXT` and `css` from outside the `or-mwc-input` component.

Reply:
I handle to get rid of this new type :-)